package.json

```diff
@@ -19,14 +19,14 @@
 		"rimraf": "^2.6.1",
 		"symlink-or-copy": "^1.2.0"
 	},
-	"babel": {
-		"presets": "@wordpress/default"
-	},
 	"jest": {
 		"collectCoverageFrom": [
 			"packages/*/**/*.js"
 		],
-		"preset": "@wordpress/jest-preset-default"
+		"preset": "@wordpress/jest-preset-default",
+		"transform": {
```

Comment:
We need to find a better way to apply this transform behind the scenes because we will have to apply a similar fix for Gutenberg otherwise.

Reply:
What's this transform about?

Reply:
I left a comment in the description:

> I have some issues with `babel-jest` which is no longer working as I would expect. I had to apply `"^.+\\.jsx?$": "<rootDir>/packages/scripts/config/babel-transform.js"` in the main `package.json` to make it work. Ideally, it shouldn't be possible at all when overriding Jest config.

The thing is Jest is still on Babel 6 and we try to make it work with Babel 7, but something is broken and I don't see Babel transforms applied properly.